functions/go/inflate-helm-chart/helminflatorplugin_test.go

```diff
@@ -1,623 +0,0 @@
-// Copyright 2019 The Kubernetes Authors.
```

Comment:
Why we had k8s license header in files outside of third_party directory?

Reply:
My mistake, I must have copied and pasted something incorrectly. I've audited the other files to check that the license headers are correct, sorry about that

Reply:
No worries. You can do that in a separate PR.